app/components/Errors.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { map } from "lodash";
+import CSSModules from "react-css-modules";
+import styles from "../styles/components/errors.css";
+
+class Errors extends React.Component {
+  render () {
+    const { errors } = this.props;
+    const errorsMap = map(errors, (error, index) => {
+      return <li key={index} styleName="list-item">{error}</li>
+    });
+
+    return (
+      <ul styleName="unordered-list">{errorsMap}</ul>
```

Comment:
This is a bad class name, try to make something more descriptive like `errors` or `errors-list`